### ISSUE TITLE
Tool crashing after game update

### ISSUE THREAD

**Issue by rustedplastic** [AngryHank/Lin2RevJunk#8]
Anyone have new CSV files after the recent 10/24 update? Can't seem to find the data.pak to extract and every time I pull up the clan member page now the tool just crashes.

**Comment by LordAzuRa** [comment]
Having the same issue here.
I get one name out of the Guild Member Info and then it crashes and the rest of the sheet doesn't populate.

**Comment by backspace123** [comment]
I don't have the new csv files after update, but to fix the member page issues, in PktGuildMemberListReadresult.cs, use `packet.Skip(24)` before writing to the file. 

I also had to change the `while (packet.Remaining > 0)` to `while (packet.Remaining > 1)` because now there is an extra byte after the last member.

That did it for me. Hope this helps!

**Comment by SundownRD** [comment]
@backspace123 So where exactly am I suppose to put that? I've tried it in the spot you said and it didn't work. WOuld you be able to post a ss of what urs looks like just to make sure? Thank you!

**Comment by backspace123** [comment]
@SundownRD forgive me..I'm bad at describing places and giving directions. Let me try again.. 

Snip of `while (packet.Remaining > 1)`:
![image](https://user-images.githubusercontent.com/44692367/47896722-bc9c4f80-dea9-11e8-93fe-0b2419fd9c6d.png)

Snip of `packet.Skip(24);`:
![image](https://user-images.githubusercontent.com/44692367/47896788-04bb7200-deaa-11e8-93d1-c333c196fc7a.png)

I hope this helps!

I'm on the SEA server, btw. I'm not sure if this will work on other servers. :|

**Comment by SundownRD** [comment]
@backspace123 Ahhh yes, I was like was I suppose to add that to the script or change one of them all together. Sadly didn't work completely, gave me more information but still popped up with the same error, btw I appreciate you coming on here and providing some information on possible fixes. Pic is of the spreadsheet, first two are unchanged. The two after are with the change. Thanks again Backspace123 and Ahh yes I am on US Have you changed anything else?
![untitledasd](https://user-images.githubusercontent.com/39049693/47897097-41ea1a80-de3e-11e8-87f9-bfb8a5fbc249.png)

**Comment by backspace123** [comment]
@SundownRD those were the only changes I did for the member list. What was the number on your packet.Skip before this? For the last few game updates on SEA, I had to change the number from 1 to 8 to 11, then 24 for the current one. You can try the previous value + 1. If it doesn't work, keep adding one until you find the sweet spot.  I thought about creating an NA account to give you the exact number, but I guess it would take longer.

**Comment by SundownRD** [comment]
@backspace123 Yeah everything broke right after this last Orc release patch. I have fixed a problem similar to this prior. But for some reason I can't fix this for the life of me. And it was set at (1)

**Comment by backspace123** [comment]
@SundownRD (and others who are not in SEA) packet.Skip for current INT (US/EU/OC) version should be 11.

**Comment by Moxer81** [comment]
@backspace123 I did all what you instructed and rebuilt the solution. Now the tool does not crash but the output folder is always empty. I see the tool detecting the packets but nothing is written into output. Is there any solution for this?

**Comment by kamertonas** [comment]
Same here im on EU packet remaining 1 skip 11 after build reader crashes instantly with error that packets folder and bat files in it not found and if you create a folder packets you see packets going, well, some of them and nothing appears in output folder.

**Comment by Moxer81** [comment]
@kamertonas Yes this exactly what happened with me. The new build requires you to create a folder named Packets so it does not crash, but nothing is written on output folder

**Comment by SundownRD** [comment]
Send me a message on discord at BootyClapital#7289 . And I will try and help you guys out. @Moxer81 and @kamertonas unless you guys have already pmed me.

**Comment by backspace123** [comment]
@kamertonas @Moxer81 
First, you need to make sure you have the correct handler in Program.cs, line 141.
Next, you need to add some lines to the Handler___.cs to generate the csv. 
For example, for the guild member list: 
`                case 1403:    
                    result = "PktGuildMemberListReadResult";
                    PktGuildMemberListReadresult.Packet(packet);` 

Hope this helps!

**Comment by ibaaM** [comment]
someone share working file

**Comment by fibroidjames** [comment]
does anyone have a full complied new version that works with the latest patches ?

**Comment by rejj04** [comment]
Please share functional version if you have one!
